pkg/result/processors/fixer.go

```diff
@@ -218,7 +218,7 @@ func (f Fixer) writeFixedFile(origFileLines [][]byte, issues []result.Issue, tmp
 		}
 
 		origFileLineNumber := i + 1
-		if nextIssue == nil || origFileLineNumber != nextIssue.Line() {
+		if nextIssue == nil || origFileLineNumber != nextIssue.GetLineRange().From {
```

Comment:
Of course there's no unit testing on this. Ugh. Because of the way `GetLineRange` defaults to the same value as `Line` if the line range isn't set, I'm mostly pretty confident this is correct. I can't find any scenario that would break because of this change in the code base (and it's pretty arguable that if such a scenario exists, it's because of intentional or persistent misuse).